_data/jobs.yml

```
@@ -1,3 +1,9 @@
+- {expires: 2021-04-01, location: 'Harvard University, Cambridge, MA', 
+    name: Research Software Engineer, url: 'https://sjobs.brassring.com/TGnewUI/Search/home/HomeWithPreLoad?partnerid=25240&siteid=5341&PageType=JobDetails&jobid=1543317#jobDetails=1543317_5341'}
+- {expires: 2021-04-01, location: 'UC San Diego', name: 'Applications Programmer',
+    url: 'https://jobs.ucsd.edu/bulletin/job.aspx?jobnum_in=106603'}
+- {expires: 2021-04-01, location: 'University of Pittsburgh'
```

```suggestion
- {expires: 2021-04-01, location: 'University of Pittsburgh',
```
We need a comma here since it's a continuation to the next line.